packages/web-twig/src/Resources/components/Dropdown/Dropdown.twig

```diff
@@ -0,0 +1,22 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _class = props.class | default(null) -%}
+{%- set _breakpoint = props.breakpoint -%}
+{%- set _elementType = props.elementType | default('div') -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'Dropdown' -%}
```

Comment:
The `fullWidth` modifier is missing here. Sorry I didn't notice before, the `web` implementation wasn't quite ready anyway.

```suggestion
{%- set _rootClassName = _spiritClassPrefix ~ 'Dropdown' -%}
{%- set _rootFullWidthClassName = _spiritClassPrefix ~ 'Dropdown--fullWidth' -%}
```

Reply:
Ok, added.